examples/nlp/parameter_efficient_finetuning_of_gpt2_with_lora.py

```diff
@@ -37,26 +37,42 @@
 ## Setup
 
 Before we start implementing the pipeline, let's install and import all the
-libraries we need. We'll be using the KerasNLP library.
-
-Secondly, let's enable mixed precision training. This will help us reduce the
-training time.
+libraries we need. We'll be using the KerasNLP library. Also, let's choose
```

Comment:
Maybe copy some language from [here](https://github.com/keras-team/keras-io/pull/1432/files#r1275615099). In particular a link to Keras Core would be good.